pkg/connector/matrix.go

```diff
@@ -166,6 +167,26 @@ func (l *LinkedInClient) HandleMatrixMessage(ctx context.Context, msg *bridgev2.
 		}
 	}
 
+	if msg.ReplyTo != nil {
+		intent := l.userLogin.Bridge.Matrix.GhostIntent(l.userID)
+		evt, err := intent.GetEvent(ctx, msg.Event.RoomID, msg.ReplyTo.MXID)
+		if err != nil {
+			zerolog.Ctx(ctx).Debug().Err(err).Stringer("romm_id", msg.Event.RoomID).Stringer("event_id", msg.ReplyTo.MXID).Msg("failed to get ReplyTo event")
+		}
+		if evt != nil {
```

Comment:
if linkedin renders replies without text in some half-sensible way, then it'd be better to preserve the reply metadata and just not provide the text when the original event can't be fetched

Reply:
Hmm, I guess it's better to just drop it then, although then the `CapLevelFullySupported` for Reply isn't actually accurate for cloud. Not sure if anything can be done about that though